labgrid/resource/remote.py

```diff
@@ -217,3 +217,20 @@ class NetworkDeditecRelais8(RemoteUSBResource):
     def __attrs_post_init__(self):
         self.timeout = 10.0
         super().__attrs_post_init__()
+
+
+@attr.s(cmp=False)
+class RemoteGPIOResource(NetworkResource, ManagedResource):
+    manager_cls = RemotePlaceManager
+
+    chip = attr.ib(validator=attr.validators.optional(attr.validators.instance_of(str)))
+    offset = attr.ib(validator=attr.validators.optional(attr.validators.instance_of(int)))
+
+
+@target_factory.reg_resource
+@attr.s(cmp=False)
+class NetworkLibGPIOD(RemoteGPIOResource):
```

Comment:
Do you have a specific reason to use two separate classes?

Reply:
I had two scenarios in mind:
1. Someone implements the deprecated sysfs interface using SSHDriver
2. People running a different OS (bsd/whatever)

Reply:
I currently don't see running labgrid on anything else than linux as a requirement.

As an aside, a driver for the sysfs interface could/would also use the agent mechanism.

Reply:
resolved by https://github.com/labgrid-project/labgrid/pull/468/commits/9e7cfbefa99551d7fdeb2ce7dc0b690d99d7bef6